service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/builder/GpcRequestBuilder.java

```diff
@@ -108,16 +116,48 @@ private ParametersParameterComponent buildParamterComponent(String parameterName
 
     public RequestHeadersSpec<?> buildGetDocumentRecordRequest(GetGpcDocumentTaskDefinition documentTaskDefinition) {
         SslContext sslContext = requestBuilderService.buildSSLContext();
-        HttpClient httpClient = HttpClient.create().secure(t -> t.sslContext(sslContext));
+        HttpClient httpClient = buildHttpClient(sslContext);
```

Comment:
@mmackiewicz1  @Broom94  this is the change that will fix the problem we found testing NIAD-820!